javascriptv3/example_code/libs/scenario/steps-common.js

```diff
@@ -44,3 +44,21 @@ export const exitOnFalse = (scenarios, stateKey) =>
       process.exit(0);
```

Comment:
Didn't we just have a conversation about not doing this?

Reply:
Yes we did. That would be another largish refactor in an already unwieldy set of changes. I'll add it in the next PR.